src/rs/server_client_test.rs

```diff
@@ -31,12 +33,24 @@ pub fn get_test_cert_hash() -> String {
             s.pop();
         }
     };
-    s
+    Credential::CertificateHash(CertificateHash::from_str(&s).unwrap())
+}
+
+/// Use test certs from openssl submodule
+#[cfg(not(target_os = "windows"))]
+pub fn get_test_cred() -> Credential {
+    use crate::CertificateFile;
+    let root_dir = std::env::current_dir().unwrap();
+    let cert_dir = root_dir.join("submodules/openssl/test/certs");
```

Comment:
I don't care for depending on a specific submodule file.

1. I want to make sure we can support openssl3 in the near future.
2. I am thinking about moving away from submodules and dynamically resolving these dependencies (probably via cmake), and they might be in different directories.

Reply:
Changed to use openssl cli to generate test certs in temp dir.